wpiutil/src/main/native/include/units/units.h

```diff
@@ -80,6 +80,10 @@
 #include <cmath>
 #include <limits>
 
+#ifndef UNIT_LIB_DISABLE_IOSTREAM
+#define UNIT_LIB_DISABLE_IOSTREAM
+#endif
+
 #if !defined(UNIT_LIB_DISABLE_IOSTREAM)
```

Comment:
So, uh, what if a team wants to turn _on_ the iostream features?

Reply:
This was to test how much faster compilation got with this removed, and on Windows it's about 25% faster. Unsure if we will go through with it, but if we do iostream will just be permanently disabled. We might look into making it compatible with raw_ostream though, as that header is much lighter weight.

What started this is we were doing some profiling, and including units.h was probably 60% of the parsing load on wpilibc.c. just the single file.

Reply:
I would recommend we instead just reverse the ifdef, change it to UNIT_LIB_ENABLE_IOSTREAM and require the user to #define it.